pkg/expr/sql_command.go

```diff
@@ -40,10 +41,13 @@ func NewSQLCommand(refID, rawSQL string) (*SQLCommand, error) {
 	if tables != nil {
 		logger.Debug("REF tables", "tables", tables, "sql", rawSQL)
 	}
+
+	defaultLimit := int64(20)
```

Comment:
Does this line and `expressions.Key("sql_expression_row_limit").MustInt64(100000)` need to be the same?

Reply:
I think I should remove this entirely. Just gonna make sure it all works properly